test/Microsoft.AspNetCore.Authentication.Test/Twitter/TwitterMiddlewareTests.cs

```diff
@@ -184,7 +184,8 @@ private static TestServer CreateServer(TwitterOptions options, Func<HttpContext,
                 })
                 .ConfigureServices(services =>
                 {
-                    services.AddAuthentication();
+                    services.AddAntiforgery();
```

Comment:
You don't need AddAntiforgery here anymore because it's in AddCookieAuthentication, right?